<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Operaciones</class>
 <widget class="QWidget" name="Operaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>731</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Recursos/images.qrc">:/operaciones-design.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="id_clientes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_productos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Productos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_proveedores">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Proveedores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_lineaproductos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Linea de productos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_trabajadores">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trabajadores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_regresarmenu">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
   <property name="icon">
    <iconset resource="../Recursos/images.qrc">
     <normaloff>:/regreso.png</normaloff>:/regreso.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos/images.qrc"/>
 </resources>
 <connections/>
</ui>
